api/routes/family.py

```diff
@@ -132,7 +132,12 @@ async def get_pedigree(
     )
 
 
-@router.get('/{project}/', operation_id='getFamilies', tags=['seqr'])
+@router.get(
+    '/{project}/',
+    operation_id='getFamilies',
+    response_model=List[Family],
```

Comment:
Response model here breaks the Python API, and it should be covered by the GraphQL API right (if not we should add it).
```suggestion
```